pkg/kubectl/history.go

```diff
@@ -153,9 +153,9 @@ type DaemonSetHistoryViewer struct {
 // ViewHistory returns a revision-to-history map as the revision history of a deployment
 // TODO: this should be a describer
 func (h *DaemonSetHistoryViewer) ViewHistory(namespace, name string, revision int64) (string, error) {
-	versionedExtensionsClient := versionedExtensionsClientV1beta1(h.c)
 	versionedAppsClient := versionedAppsClientV1beta1(h.c)
-	ds, allHistory, err := controlledHistories(versionedExtensionsClient, versionedAppsClient, namespace, name)
+	versionedExtensionsClient := versionedExtensionsClientV1beta1(h.c)
```

Comment:
I switched the order that puts `versionedExtensionsClient` before `versionedAppsClient`. There are 2 reasons behind:
(1) it is alphabetically ordered for having `versionedAppsClient` before `versionedExtensionsClient`
(2) `controlledHistories` function brings argument in the order of `versionedAppsClient` and then `versionedExtensionsClient`